docs/pages/reference/backends.mdx

```diff
@@ -42,53 +42,51 @@ run in a highly available fashion.
 
 ## Auth Server State
 
-To run multiple instances of Teleport Auth Server, you must switch to a High Availability secrets back-end first.
+To run multiple instances of Teleport Auth Server, you must switch to a High
+Availability secrets back-end first.
```

Comment:
```suggestion
To run multiple instances of Teleport Auth Server, you must switch to a high-availability secrets backend first.
```
- high availability is not a proper noun, so it technically shouldn't be capitalized (though I do see it capitalized elsewhere on this page, so it's probably fine to leave it capitalized here for consistency)
- the rest of the page uses "backend"